tfx/components/schema_gen/executor.py

```diff
@@ -89,5 +90,10 @@ def Do(self, input_dict: Dict[str, List[types.Artifact]],
         artifact_utils.get_single_uri(
             output_dict[standard_component_specs.SCHEMA_KEY]),
         DEFAULT_FILE_NAME)
+
+    # Add tensor representations to handle SequenceExamples downstream. Still need correct Payload Format.
+    tensor_representations = tensor_representation_util.InferTensorRepresentationsFromSchema(schema)
+    tensor_representation_util.SetTensorRepresentationsInSchema(schema, tensor_representations)
```

Comment:
Can you make it formatted in `Pyink` Python formatter for consistency? (https://github.com/google/pyink)

Reply:
I have attempted to do this, although perhaps the imports need changing for this same reason? I am unsure how one recommends to do this if the updated state is not desired, as I did not come up with the module names nor function names?